src/server.h

```diff
@@ -2563,9 +2562,17 @@ int validateProcTitleTemplate(const char *template);
 int serverCommunicateSystemd(const char *sd_notify_msg);
 void serverSetCpuAffinity(const char *cpulist);
 
+/* ERROR STATS constants */
+#define ERROR_STATS_LUA_LIMIT 128 /* After the errors RAX reaches this limit, instead of tracking 
+                                    custom LUA errors, we track the error under `error_LUA_ERRORSTATS_OVERFLOW`. */
```

Comment:
nit: it is `errorstat_LUA_ERRORSTATS_OVERFLOW`. You can avoid the actual string and refer to the symbol defined below.

Reply:
Sure. Updating the comment to:
```
#define ERROR_STATS_LUA_LIMIT 128 /* After the errors RAX reaches this limit, instead of tracking 
                                    custom LUA errors, we track the error under the symbol below. */
#define LUA_ERRORSTATS_OVERFLOW_ERR "LUA_ERRORSTATS_OVERFLOW"
```